Topics/Teamwork.md

```diff
@@ -12,3 +12,25 @@ An important first step towards conflict resolution is learning how to avoid the
 [Resolving Conflicts](https://ca.indeed.com/career-advice/career-development/conflict-resolution-at-work)
 
 It is not always possible to avoid conflict. In work environment when people are exchanging ideas and working towards goals, a difference of opinion is inevitable. However, if handled properly, these conflicts can be resolved while keeping the negative impacts at a minimum, and even help create a better solution for the project at hand. This article covers ways that the parties in a conflict can help each other understand their own points of view, while keeping discourse respectful. It emphasizes ways that team members can help each other work towards a common goal.
+
+
+
+
+
+## **Roles in a Software Team **
+
+1. Project Manager: The project manager is responsible for planning and overseeing the software development process. They set goals and deadlines, allocate resources, manage the budget, and communicate with stakeholders.
+2. Software Developer: The software developer writes, tests, and maintains the software code. They work with the project manager and other team members to ensure that the software meets the project requirements and is delivered on time.
+3. Quality Assurance (QA) Analyst: The QA analyst tests the software to identify bugs and ensure that it meets the quality standards set by the project manager. They work with the software developers to resolve any issues that arise.
+4. User Experience (UX) Designer: The UX designer creates the user interface and experience for the software. They work with the software developers to ensure that the user interface is intuitive, attractive, and easy to use.
```

Comment:
I think "The UX designer creates the user interface *and experience* for the software" sounds a bit awkward and doesn't flow naturally.